packages/protoplugin/src/create-es-plugin.ts

```diff
@@ -242,6 +249,21 @@ function parseParameter(
         rewriteImports.push({ pattern, target });
         break;
       }
+      case "keep_empty_files": {
```

Comment:
Could we shorten this to:

```js
case "keep_empty_files": {
   keepEmptyFiles = value === "true" || "1"
}
```

Reply:
It would need to be `value == "true" || value === "1"`. We _could_ do that, but I think it's better to be exact here. For example `keep_empty_files=no` should error.